README.md

```diff
@@ -37,7 +45,14 @@ yarn add @unstoppabledomains/resolution
 npm install @unstoppabledomains/resolution --save
 ```
 
-If you're interested in resolving domains via the command line, see our [CLI section](#command-line-interface). 
+If you're interested in resolving domains via the command line, see our
+[CLI section](#command-line-interface).
+
+## Obtaining JSON RPC Provider
+
+Resolution for .crypto and .eth perform reads on the Ethereum blockchain and
```

Comment:
```suggestion
Resolution for .crypto and .eth performs reads on the Ethereum blockchain and
```
I'm always unconfident when correcting English grammar, but I think it's singular here :)

Reply:
You're right! English is hard haha